filters-share/adservice.txt

```diff
@@ -115,7 +115,6 @@
 ||ad.adjw.co.kr^
 ||compass.adop.cc^
 ||catad5959.com^
-||adex.ednplus.com^
```

Comment:
This is used on https://www.chosun.com/politics/politics_general/2021/09/20/KBX3Y25ZRJF55CRPEKWEWXJNCY/.

Reply:
I know. It still exists on https://github.com/iam-py-test/List-KR/blob/master/filters-share/adservice.txt#L90 (that entry is there twice)